graphql/database/experiments/growthbookUtils.js

```diff
@@ -35,6 +38,7 @@ export const getConfiguredGrowthbook = ({
   email,
   internalExperimentOverrides = {},
   tabs,
+  noUserAttributes = false,
```

Comment:
nit: I think the name `noUserAttributes` is a bit confusing. Could we find a way to avoid double negatives and change the param name to like... `userAgnostic` or `userSpecific` (the value for `userSpecific` being default true)?

Reply:
Thanks! Changed.